src/middleware/authentication.middleware.ts

```diff
@@ -11,7 +11,8 @@ import {
 export const authentication = (req: Request, res: Response, next: NextFunction): void => {
   try {
     if (!checkUserSignedIn(req.session)) {
-      logger.infoRequest(req, 'User not authenticated, status_code=401, redirecting to sign in page');
+      logger.info("User not authenticated, status_code=401");
+      logger.infoRequest(req, 'redirecting to sign in page, status_code=302');
```

Comment:
I'm a bit confused - line 14 log message says status code is 401 and then this line says it's 302. Nothing's happened in between the two log messages so which one is right? Or, is the second one saying that the response/status code will be 302 because of the redirect? Can log messages be corrected/reworded to make clear what's going on here please?

Reply:
Both events happens and it is important to log them both. Do you have any suggestions?

Reply:
As you suggested I will remove the 401 and add one single comments with the following msg error `User not authenticated, redirecting to sign in page, status_code=302`